packages/orbit-components/src/TextLink/types.d.ts

```diff
@@ -4,10 +4,11 @@
 import type * as React from "react";
 
 import type * as Common from "../common/types";
+import type { DownloadWithHrefConditionalProps } from "../primitives/ButtonPrimitive/types";
 
 export type Type = "primary" | "secondary" | "info" | "success" | "warning" | "critical" | "white";
 
-export interface Props extends Common.Globals {
+interface IProps extends Common.Globals {
```

Comment:
do not use this convention with interfaces, rewrite it into something like `CommonProps`, `BaseProps`, `Properties` etc.

Reply:
Just FYI. The Typescript Handbook tells the developer not to prefix interfaces with I.